app/src/main/java/org/mozilla/fenix/settings/creditcards/CreditCardEditorFragment.kt

```diff
@@ -82,10 +91,12 @@ class CreditCardEditorFragment : SecureFragment(R.layout.fragment_credit_card_ed
     }
 
     /**
-     * Close any open dialogs or menus and reauthenticate if the fragment is paused and
-     * the user is not navigating to [CreditCardsManagementFragment].
+     * Close the keyboard, any open dialogs or menus and then reauthenticate if the
+     * fragment is paused and the user is not navigating to
+     * [CreditCardsManagementFragment].
```

Comment:
```suggestion
     * fragment is paused and the user is not navigating to [CreditCardsManagementFragment].
```

Reply:
Yup, sorry about that. Fixed up the original commit with the error and force pushed. Thanks!